magento/ece-tools
# MAGECLOUD-1057: Add possibility to change SCD strategy

https://magento2.atlassian.net/browse/MAGECLOUD-1057

### Description


### Fixed Issues (if relevant)

1. Added `-s` option for `build` and `deploy` phases
2. Added `SCD_STRATEGY` variable for build and deploy
3. Cleaned `File.php` class
4. Added integration test to verify deploy static in `build` phase
5. Added lazy loading of environment data

### Manual testing scenarios

1. Set `SCD_STRATEGY` variable to build or deploy phase
2. Deploy static content
3. Check if Magento works

### Contribution checklist
 - [ ] Pull request was approved by architect
